src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -5199,6 +5199,36 @@ private BoundExpression BindCollectionExpression(CollectionExpressionSyntax synt
 
             MessageID.IDS_FeatureCollectionExpressions.CheckFeatureAvailability(diagnostics, syntax, syntax.OpenBracketToken.GetLocation());
 
+            for (int i = 1, n = syntax.Elements.Count - 1; i < n; i++)
+            {
+                var element = syntax.Elements[i];
+                if (element is WithElementSyntax withElement)
+                {
+                    Error(diagnostics, ErrorCode.ERR_WithElementMustBeFirst, withElement.WithKeyword.GetLocation());
```

Comment:
>ERR_WithElementMustBeFirst [](http://example.com/codeflow?start=49&length=26)

Are we testing this?

Reply:
likely not. t his was just the parsing PR.  binding/testing was intended for you to do :)

Reply:
If we're adding an error code and binding logic, we should test it. Either way seems fine for this PR (remove logic or test)